py/mpconfig.h

```diff
@@ -490,6 +490,11 @@
 #define MICROPY_LONGINT_IMPL (MICROPY_LONGINT_IMPL_NONE)
 #endif
 
+// Support for calls to buildin.pow() with 3 integer arguments
+#ifndef MICROPY_INT_POW3
+#define MICROPY_INT_POW3 (1)
```

Comment:
Note that the final decision is after BDFL. (But we all hope that he won't whimsically introduce [math.tau's](https://www.python.org/dev/peps/pep-0628/) ;-). )

> Perhaps we could put the minimal Python implementation into the text of the NotImplemented exception :-)

Curb your enthusiasm ;-). One issue I didn't mention is that error messages you introduced are very, very long and eat precious bytes! When nobody's looking, I'm going to curb them down to bare minimum (which is often `""`) ;-). Seriously speaking, we have framework to manage that: https://github.com/micropython/micropython/blob/master/py/mpconfig.h#L513 , but as someone who's a configurability geek, I can say that it does cost effort, so it's better to have a single good default (which in uPy case translates to be conservatively short/small).